src/transformers/models/beit/configuration_beit.py

```diff
@@ -132,7 +132,7 @@ def __init__(
         image_size=224,
         patch_size=16,
         num_channels=3,
-        use_mask_token=False,
+        use_mask_token=True,
```

Comment:
Why change this?

Reply:
Oh that was done to fix the embedding error, I forgot to revert it

Reply:
And because if the masked token is not used, the bool_masked_pos becomes a zeros array and the embedding test fails

Reply:
I fixed it by removing the if condition from the model itself